artwork/linedrawings/aaron.jsx

```diff
@@ -0,0 +1,182 @@
+import React from "react";
+
+function Icon() {

```

Comment:
We only use named exports in v3. We also want to allow people to override the styling, so the signature should be as follows:

export const Aaron = () => {
  // svg here
}